packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -158,8 +159,9 @@ export class LoaderContainerTracker implements IOpProcessingController {
      *      - this overlaps with !isDirty, but include task scheduler ops.
      *      - Trailing NoOp is tracked and don't count as pending ops.
      */
-    public async ensureSynchronized(...containers: IContainer[]) {
+    public async ensureSynchronized(timeoutDuration = defaultTimeoutDurationMs, ...containers: IContainer[]) {
```

Comment:
i worry using defaultTimeoutDurationMs will cause test failures in our e2e tests as it is a very low value by default, 250ms i think